src/Symfony/Component/Form/DataTransformerInterface.php

```diff
@@ -17,6 +17,9 @@
  * Transforms a value between different representations.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @template T
+ * @template R
```

Comment:
I would prefer to follow Psalm's own convention and use slightly more describtive template names, e.g:

```suggestion
 * @template TValue
 * @template TTransformedValue
```

Reply:
Done